src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -0,0 +1,79 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers.js';
+import { createClass } from '../../util/component-types.js';
+
+const cx = lucidClassNames.bind('&-ProgressBar');
+
+const { number, string, oneOf, any, bool } = PropTypes;
+
+const ProgressBar = createClass({
+	displayName: 'Progress Bar',
+
+	statics: {
+		peek: {
+			description: `
+				A Progress Bar component is used to indicate progress in a procedure
+				consisting of numerous discrete steps or continuous operation.
+			`,
+
+			categories: ['communication'],
+		},
+	},
+
+	propTypes: {
+		className: string`
+			Appended to the component-specific class names set on the root element.
+		`,
+
+		kind: oneOf(['default', 'success', 'danger', 'info', 'warning'])`
+			Applies a color style for the kind of ProgressBar.
+		`,
+
+		percentComplete: number`
+			Percentage ProgressBar is complete.
+		`,
+
+		children: any,
+
+		hasTitle: bool`
+			Whether or not a title is displayed above the ProgressBar.
+		`,
+
+		title: string`
```

Comment:
Let's change the `title` prop to be a `Title` prop and allow folks to pass more than just text. 

Here's an example of how we did it with `ToolTip`: https://github.com/appnexus/lucid/blob/c57a348669a15fa727865241d0e4bdee0912e28e/src/components/ToolTip/ToolTip.jsx#L118